tests/unit/converters/test_vision_excel_converter.py

```diff
@@ -139,3 +140,10 @@ def test_get_appliance_id(converter: VisionExcelConverter):
 
     with pytest.raises(KeyError):
         converter.get_appliance_id(table="Sources", node_number=1, sub_number=3)
+
+
+def test_ambiguity_in_vision_excel():
+    test_file = Path(__file__).parents[2] / "data" / "vision" / "data.xlsx"
```

Comment:
data.xlsx is directly in a crowded data / vision folder. A rename would be nice.

```suggestion
    excel_file_checker = ExcelAmbiguityChecker("excel_ambiguity_check_data.xlsx")
```